src/resources/views/character/mail-timeline.blade.php

```diff
@@ -5,6 +5,7 @@
 
 @inject('CharacterInfo', 'Seat\Eveapi\Models\Character\CharacterInfo')
 @inject('CorporationInfo', 'Seat\Eveapi\Models\Corporation\CorporationInfo')
+@inject('MailMailingList', 'Seat\Eveapi\Models\Mail\MailMailingList')
```

Comment:
Will make some tries tomorrow, but I don't like those Inject :p
We should avoid them as much as possible.

Reply:
i know how you feel ... the whole blade does live up to my expectations. However because there were quick fixes to make it usable, i went for those low hanging fruits, until we manage to refactor the whole mail-timeline.